From 76b0be64d14fed5f6d6c5c71474d19298a86005e Mon Sep 17 00:00:00 2001
From: w08r <will@hazy.com>
Date: Fri, 15 Dec 2023 08:57:21 +0000
Subject: [PATCH] Add cstdint header to SnowFlakeType.hpp

Not quite sure how this compiled before, but this is required for use
of the uint8_t type in the SnowflakeType::Type enum definition.

Fixes 1823.
---
 .../connector/nanoarrow_cpp/ArrowIterator/SnowflakeType.hpp      | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/snowflake/connector/nanoarrow_cpp/ArrowIterator/SnowflakeType.hpp b/src/snowflake/connector/nanoarrow_cpp/ArrowIterator/SnowflakeType.hpp
index c9204d5d0..b6150b26b 100644
--- a/src/snowflake/connector/nanoarrow_cpp/ArrowIterator/SnowflakeType.hpp
+++ b/src/snowflake/connector/nanoarrow_cpp/ArrowIterator/SnowflakeType.hpp
@@ -5,6 +5,7 @@
 #ifndef PC_SNOWFLAKETYPE_HPP
 #define PC_SNOWFLAKETYPE_HPP
 
+#include <cstdint>
 #include <string>
 #include <algorithm>
 #include <unordered_map>
